src/components/ArrowLink/ArrowLink.module.scss

```diff
@@ -16,7 +16,7 @@
 
 .ArrowContainer {
 	position: absolute;
-	left: calc(100% + 5px);
+	left: -12%;
```

Comment:
Hey @colbr, I've reverted the changes here. When applying ```back``` to the arrow link this seems to just spin the arrow 180° rather than place the arrow to the left of the text. Not sure if I'm missing something there but I've tweaked this so if ```back``` is applied, it will place the arrow to the left of the text. Seems to work ok here and in other instances its used but let me know if this needs changing.